### ISSUE TITLE
error getting ClusterInformation

### ISSUE THREAD

**Issue by agvaliente** [projectcalico/calico#4095]
Hello

I have installed calico on premise and my pod calico-kube-controllers-c9784d67d-hddbs reports this error:

  Warning  FailedCreatePodSandBox  23s    kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create pod network sandbox k8s_calico-kube-controllers-c9784d67d-dr6tx_kube-system_37151347-8808-4b21-b20f-6a1edf71899e_0(0a75fd5749c304700094369a49b568ec7971124c56187dbaff6823ddb087233e): error getting ClusterInformation: Get "https://[10.96.0.1]:443/apis/crd.projectcalico.org/v1/clusterinformations/default": unexpected EOF

If I rename the file /etc/cni/net.d/10-calico.conflist the pod starts correcly and doesn´t report errors.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

My file has this content:

{
  "name": "k8s-pod-network",
  "cniVersion": "0.3.1",
  "plugins": [
    {
      "type": "calico",
      "log_level": "info",
      "log_file_path": "/var/log/calico/cni/cni.log",
      "datastore_type": "kubernetes",
      "mtu": 1440,
      "ipam": {
          "type": "calico-ipam"
      },
      "policy": {
          "type": "k8s"
      },
      "kubernetes": {
          "kubeconfig": "/etc/cni/net.d/calico-kubeconfig"
      }
    },
    {
      "type": "portmap",
      "snat": true,
      "capabilities": {"portMappings": true}
    },
    {
      "type": "bandwidth",
      "capabilities": {"bandwidth": true}
    }
  ]
}

**Comment by agvaliente** [comment]
The problem was resolved by configuring the env no_proxy: 10.96.0.1 in crio and kubelet configuration

systemctl edit crio
systemctl edit kubelet

**Comment by danifr** [comment]
Just for the record In y case I needed to follow: https://docs.projectcalico.org/maintenance/troubleshoot/troubleshooting#configure-networkmanager

--------------------------

**Configure NetworkManager**

Configure NetworkManager before attempting to use Calico networking.

NetworkManager manipulates the routing table for interfaces in the default network namespace where Calico veth pairs are anchored for connections to containers. This can interfere with the Calico agent’s ability to route correctly.

Create the following configuration file at /etc/NetworkManager/conf.d/calico.conf to prevent NetworkManager from interfering with the interfaces:

```
[keyfile]
unmanaged-devices=interface-name:cali*;interface-name:tunl*;interface-name:vxlan.calico
```
